composer.json

```diff
@@ -16,9 +16,12 @@
   "require-dev": {
     "doctrine/coding-standard": "^4.0",
     "phpunit/phpunit": "^7.2",
-    "psr/http-message": "^1.0"
+    "phpstan/phpstan-shim": "^0.10",
+    "psr/http-message": "^1.0",
+    "react/promise": "2.*"
```

Comment:
I'd also add `composer require --dev phpstan/phpstan-phpunit`

Reply:
We using this only for src in this case so no need an extension for phpunit

Reply:
Followup https://github.com/webonyx/graphql-php/pull/333#pullrequestreview-150080275 First let's resolve that